micrometer-binders/build.gradle

```diff
@@ -0,0 +1,158 @@
+plugins {
+    id 'idea'
+}
+
+dependencies {
+    // TODO(anuraaga): HdrHistogram is exposed in the micrometer API but probably shouldn't be
+    api 'org.hdrhistogram:HdrHistogram'
+    implementation('org.latencyutils:LatencyUtils') {
+        exclude group: 'org.hdrhistogram', module: 'HdrHistogram'
+    }
+
+    api project(':micrometer-core')
```

Comment:
They will need both dependencies on their classpath, but a dependency on `micrometer-binders` will have a transitive dependency on `micrometer-core`, so if they want, they can declare the one dependency instead of both.